geo-postgis/Cargo.toml

```
@@ -11,5 +11,5 @@ description = "Conversion between `geo-types` and `postgis` types."
 edition = "2018"
 
 [dependencies]
-postgis = "0.7"
+postgis = { version = ">=0.7.0, <9.0" }
```

```suggestion
postgis = { version = ">=0.7.0, <0.9.0" }
```

TBH I understand, but don't like this style of pinning, since some of the crates that use it are very slow at putting up new releases.

---

Wow, bad typo on my part. Thank you for noticing! ❤️ 

> TBH I understand, but don't like this style of pinning, since some of the crates that use it are very slow at putting up new releases.

Interesting, but I'm not sure I understand.

My (simple) rule is to avoid unnecessary breaking changes. Requiring postgis 0.8 would be an unnecessary breaking change, so we'd have to breaking-bump semver, and anyone downstream that depends on this crate would have to breaking-bump semver on their crate.

Can you give an example or clarify why it might be better to force this cascade of updates to happen when we could instead support both the new and the old?

---

No, it's fine to support both `0.7` and `0.8`, just that (as a user of crates that do this -- `diesel` comes to mind), when `0.9` comes out, I won't be able to upgrade to it until `postgis` changes the dependency to `>= 0.7, < 0.10`, which might take a long time to get in a release.

So it's correct to do this, just very annoying at times.